src/sentry/data_export/tasks.py

```diff
@@ -218,18 +230,14 @@ def process_rows(processor, data_export, batch_size, offset):
 @handle_snuba_errors(logger)
 def process_issues_by_tag(processor, limit, offset):
     gtv_list_unicode = processor.get_serialized_data(limit=limit, offset=offset)
-    # TODO(python3): Remove next line once the 'csv' module has been updated to Python 3
-    # See associated comment in './utils.py'
-    gtv_list = convert_to_utf8(gtv_list_unicode)
+    gtv_list = ensure_unicode_for_csv(gtv_list_unicode)
     return gtv_list
 
 
 @handle_snuba_errors(logger)
 def process_discover(processor, limit, offset):
     raw_data_unicode = processor.data_fn(limit=limit, offset=offset)["data"]
-    # TODO(python3): Remove next line once the 'csv' module has been updated to Python 3
-    # See associated comment in './utils.py'
```

Comment:
Shouldn't this comment still be here? I think the point is that this line can be removed when we're entirely on py3. Alternatively we could have a comment like this in `ensure_unicode_for_csv`

Reply:
I think I left a comment like this in the function itself